Makefile

```diff
@@ -166,6 +166,7 @@ generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 .PHONY: release-manifests
 release-manifests: ## Builds the manifests to publish with a release
 	@mkdir -p out
+	cd config/manager/; ../../$(KUSTOMIZE) edit set image gcr.io/cluster-api-provider-vsphere/release/manager=gcr.io/cluster-api-provider-vsphere/release/manager:$(VERSION)
```

Comment:
need to error if VERSION isn't set

Reply:
Don't have much experience with kustomize, but I think this works too and seems cleaner:
```
cd config/manager/; ../../$(KUSTOMIZE) edit set image gcr.io/cluster-api-provider-vsphere/release/managerr:$(VERSION)
```